influxdb/tasks/install.yml

```diff
@@ -1,11 +1,12 @@
 ---
 - name: Download InfluxDB
+  when: not influxdb_retrieve_from_s3 | default(false)
```

Comment:
Call the variable `influxdb_download` and default it to `true`, then this line can just be `when: influxdb_download`